testWork1/task1.c

```diff
@@ -0,0 +1,140 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+// #define DEBUG 1
+
+#define arrType int
+
+void swap(arrType *a, arrType *b)
+{
+    const arrType c = *a;
+    *a = *b;
+    *b = c;
+}
+
+void insertionSort(size_t len, arrType *arr)
+{
+    for (size_t i = 0; i < len; ++i)
```

Comment:
```suggestion
    for (size_t i = 1; i < len; ++i)
```
Зачем выполнять дополнительную итерацию, которая гарантировано ничего не сделает?